storm-core/test/jvm/org/apache/storm/windowing/WaterMarkEventGeneratorTest.java

```diff
@@ -56,6 +57,13 @@ public void add(Event<Integer> event) {
         waterMarkEventGenerator.start();
     }
 
+    @After
+    public void tearDown() {
+        if (waterMarkEventGenerator != null) {
```

Comment:
Nit: I don't think this can be null

Reply:
Yes I just add it as defensive progressing, but can be removed. I'll address.